### ISSUE TITLE
Add onlyfans.com

### ISSUE THREAD

**Issue by WebBreacher** [WebBreacher/WhatsMyName#226]
http://onlyfans.com/

PORN

https://onlyfans.com/baberose
https://onlyfans.com/melrosefox
https://onlyfans.com/nena_ko

**Comment by swedishmike** [comment]
I had a quick look at adding this but it seems that it whinges about lack of JavaScript and don't give you a 'properly rendered' page without it.

There's no difference in result code between success and failure due to this either. 

Not sure if there's any other way to get around this?

**Comment by WebBreacher** [comment]
hmm....sorry for the delayed response. really busy now and cannot look further at this. can keep this here but wait a bit.

**Comment by djahren** [comment]
> There's no difference in result code between success and failure due to this either.
> Not sure if there's any other way to get around this?

I've found the same thing. The HTML is identical and the site content is loaded through JavaScript. I was able to look at the resources called when loading the page and it does call their internal API so I had a glimmer of hope as https://onlyfans.com/api2/v2/users/baberose gets loaded with the page, however, navigating to the URI outside of the site elicits an API error whether it's a valid user or not, meaning they likely are checking the origin of the request and returning an error if it's not internal. 

Unless there's a different API that doesn't do origin checking, I don't think it's possible to check OF users via this method, however I don't think that's likely.
